### PR TITLE
New form template

### PR DESCRIPTION
**Pull request by njseeto**
[Trello](https://trello.com/c/G4HR2IxT/2117-change-new-form-default-template-to-have-cya-and-confirmation-5-7)
### New form template 
This adds the CYA and Confirmation pages to the default service template.
This is hidden behind a feature flag so will only appear once branching is released, this is because we require the connection menus to be present.

### Update Acceptance tests 
Tests have been updated to accommodate the new service default template.

### CODE REVIEW

**Review by tomas-stefano — APPROVED** [review]

File: app/generators/new_service_generator.rb [hunk 1]
```diff
@@ -46,4 +47,47 @@ def footer_pages
...
+          'default': ''
+        }
+      }
+    }
+  end
+
+  def add_cya_and_confirmation(metadata)
+    cya_metadata = (DefaultMetadata['page.checkanswers'])
+    .merge('_uuid' => SecureRandom.uuid)
+    .merge('url' => 'checkanswers')
```
Comment (brenetic) [inline app/generators/new_service_generator.rb:81]: little bit difficult to read this syntactically. can we indent the merges please.

also there is no need to wrap the call to `DefaultMetadata` in parenthesis

File: app/generators/new_service_generator.rb [hunk 2]
```diff
@@ -46,4 +47,47 @@ def footer_pages
...
+
+    confirmation_metadata = (DefaultMetadata['page.confirmation'])
+    .merge('_uuid' => SecureRandom.uuid)
+    .merge('url' => 'confirmation')
```
Comment (brenetic) [inline app/generators/new_service_generator.rb:85]: same as my comment above

**Comment by brenetic** [comment]
great stuff, nat. couple of small comments

**Review by brenetic — APPROVED** [review]
🚀

**Pull request merged**